packages/docs/content/foundation/typography/list.mdx

```diff
@@ -41,12 +46,6 @@ import loremIpsumGenerator from '../../../src/helpers/loremIpsumGenerator';
     </ol>
 </EmbeddedExample>
 
-<Alert variation="warn" weight="lightweight" class="ds-u-margin-y--3">
-  <p class="ds-u-margin-top--0">It's worth mentioning that when <code>list-style</code> is removed from a list, <a href="https://developer.mozilla.org/en-US/docs/Web/CSS/list-style#accessibility_concerns" target="_blank">Safari will not recognize it as a list</a> in the accessibility tree.</p>
-
-  <p class="ds-u-margin-bottom--0">To resolve this issue, <code>role="list"</code> should be added to a list where the styles have been removed.</p>
-</Alert>
```

Comment:
Why was this removed?

Reply:
It was added in https://github.com/CMSgov/design-system/pull/2009

Reply:
We have it down under the accessibility guidance section already. 
"When using an unstyled list, `role="list"` should be added to the list element." 

I'm also still thinking about if we should keep our examples super clean and free from extra guidance and utilize the guidance section for that purpose?  

Maybe the alert moves down to under the a11y section?

Reply:
Didn't realize it was duplicate info 👍

Reply:
Maybe we should add this part to the guidance below for a little explanation
```
<a href="https://developer.mozilla.org/en-US/docs/Web/CSS/list-style#accessibility_concerns" target="_blank">Safari will not recognize it as a list</a> in the accessibility tree.</p>
```